tests/unit_tests/test_rectangle.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
```

Comment:
See the above comment for a similar change

Reply:
done